framework/src/org/checkerframework/framework/type/SupertypeFinder.java

```diff
@@ -215,7 +215,13 @@
                 AnnotatedDeclaredType dt =
                         (AnnotatedDeclaredType) atypeFactory.toAnnotatedType(superClass, false);
                 supertypes.add(dt);
-
+                // CZC: always try to apply defaulting to the type arguments
```

Comment:
Remove the `CZC` part and you're not trying to do anything - you are applying the defaults.
So this comment should be something like `// add computed type annotations to the type arguments of the super type`